src/sth_server.js

```diff
@@ -560,7 +561,8 @@
                   contextElement = contextResponses[i].contextElement;
                   attributes = contextElement.attributes;
                   for (var j = 0; j < attributes.length; j++) {
-                    if (!attributes[j].value || isNaN(attributes[j].value)) {
+                    if (!attributes[j].value ||
+                      (isNaN(attributes[j].value) && typeof(attributes[j].value) !== 'string')) {
```

Comment:
Yeah, that's absolutely right!!! :) In fact, the guiding criteria is the `isNaN()` function... Consequently, if anything is a value which can be converted to a number (`float` via `parseFloat()`) then it will be dealt as a numeric value and aggregated correspondly, no matter what's included in the attribute type value :)